app/components/BetsListPanel.tsx

```diff
@@ -55,6 +62,9 @@ export const BetsListPanel = ({
         bets.map((userBet: UserBet) => (
           <CardComponent userBet={userBet} key={userBet.id} />
         ))}
+      {page && (page !== 1 || hasNextPage) && setPage && (
+        <Pagination isFinalPage={!hasNextPage} page={page} setPage={setPage} />
+      )}
```

Comment:
we can embed this logic inside of the component since it has all the props to make it happen, or I'm missing something?